src/components/Inputs/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -19,7 +19,7 @@ export default {
   },
   args: {
     features: [...DEFAULT_FEATURES],
-    onImageUpload: async (file: File) => {
+    onImageUpload: (file: File) => {
```

Comment:
I recently changed the type of `onImageUpload` in this component, have you merged in the newest version of main into your branch?

Reply:
yea, but if I don't remove the async tag it's throws an error because it isn't waiting on anything.

Reply:
But psuedo-'uploading' an image in the storybook works as expected? It should just show the file picker and console log the file you chose

Reply:
it seems to work the same with or without async